src/components/ButtonPanel.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Button from './Button';
+import './ButtonPanel.scss';
+
+const ButtonPanel = () => (
+  // <div className="buttonPanel" onClick={handleClick} onKeyDown={handleKeyDown}>
+  <div className="buttonPanel">
+    <div className="group1">
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" />
+    </div>
+    <div className="group2">
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="X" />
+    </div>
+    <div className="group3">
+      <Button name="4" />
+      <Button name="5" />
+      <Button name="6" />
```

Comment:
### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.